src/components/Form/Form.stories.tsx

```diff
@@ -92,6 +97,15 @@ export const Default: Story = {
 					inputType: 'tel',
 				},
 			},
+			logoImage: {
+				'ui:options': {
+					accept: {
+						'image/png': ['.png'],
+						'text/plain': ['.txt'],
```

Comment:
Why `.txt` for a logo image? 😓 
If we want to show that we can accept text documents, we should do so in another form field

Reply:
@myarmolinsky this was for basically showing the different icon when you upload something which is not an image. What do you think of changing logoImage for "asset" which can be a .png or .txt ?

Reply:
Perhaps just an additonal field for "Other documents" so that we can showcase that you can have a dropzone with limited file types permitted and one with mixed file types permitted

Reply:
@myarmolinsky Not sure, I think the overall example in the storybook looks a bit odd with two Dropzone components. Maybe we can have just one with "additional documents" which accepts png/txt ?

Reply:
Sure, let's do that, keep it simple for now
If we want we can add a separate story to showcase the FileWidget later